OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -18,7 +18,7 @@ namespace OpenRA.Mods.Common.Traits
 {
 	public enum ExplosionType { Footprint, CenterPosition }
 
-	public enum DamageSource { Self, Killer }
+	public enum DamageSource { Self, Parent, Killer }
```

Comment:
Rename `Killer` to `Target` or `Target`? Seems more in line with terminologie. Would also give option to gain XP on damage.

Reply:
This is scope creep. This PR doesn't use update rules